test/org/rascalmpl/test/util/library/RunRascalTestModules.java

```diff
@@ -0,0 +1,10 @@
+package org.rascalmpl.test.util.library;
+
+import org.junit.runner.RunWith;
+import org.rascalmpl.test.infrastructure.RascalJUnitTestPrefix;
+import org.rascalmpl.test.infrastructure.RascalJUnitTestRunner;
+
+@RunWith(RascalJUnitTestRunner.class)
+@RascalJUnitTestPrefix("lang::rascal::tests::library")
+public class RunRascalTestModules { }
```

Comment:
why is this in this PR?

Reply:
to be able to run the relevant tests (and skip all the others) that are touched by this PR.